src/components/Slider/_Slider.scss

```diff
@@ -0,0 +1,183 @@
+$light-primary: hsl(162, 73%, 70%) !default;
+$light-secondary: hsl(208, 7%, 70%) !default;
+
+.slider-cont {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.slider-vertical {
+  transform: rotate(90deg);
```

Comment:
Rotating by 270deg would allow the vertical slider to have the min value at the bottom and the max value at the top of the slider instead of the other way around.

Reply:
fixed